tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -71,3 +81,13 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
 
   verify()
 }
+
+object QualificationArgs {
+  def isOrderAsc(order: String): Boolean = {
+    order.toLowerCase.startsWith("asc")
```

Comment:
Nit: Should we have "equals" instead of "startsWith" here? 
Or the error message could be updated ""Error, the order must either be desc or asc" above. 
I am fine merging the current commit as well as this is minor.

Reply:
I intentionally left it as startsWith in case someone actually typed ascending it would still match and work.